crates/octez/src/async/protocol.rs

```diff
@@ -1,5 +1,6 @@
-pub use super::bootstrap::BootstrapAccount;
-use super::bootstrap::BootstrapAccounts;
+pub use super::bootstrap::{BootstrapAccount, BootstrapContract};
```

Comment:
I don't think so. Users only interact with `ProtocolParamsBuilder` and its method `add_contract` that involves `BootstrapContract`. The container types are mostly for managing internal logic like conflicts in contract addresses.

Reply:
Yes, that's what I meant. We could indeed remove the `pub` here and merge the imports into line 2.

Reply:
Sorry I misunderstood this a bit. I'm re-exporting `BootstrapAccount` and `BootstrapContract` here because I think it makes sense to make them available through this module. These types are both used to construct protocol parameters. Module `bootstrap` is only visible to the crate and thus `BootstrapAccount` and `BootstrapContract` won't be available through that module. I think it's okay to keep it private.

Reply:
Ah ok